torch/linalg/__init__.py

```diff
@@ -890,6 +890,129 @@
             [1, 2, 2, 2]])
 """)
 
+norm = _add_docstr(_linalg.linalg_norm, r"""
+linalg.norm(A, ord=None, dim=None, keepdim=False, *, out=None, dtype=None) -> Tensor
+
+Computes a vector or a matrix norm.
+
+If :attr:`A` is complex valued, it computes the norm of :attr:`A`\ `.abs()`
+
+Supports inputs of float, double, cfloat and cdouble dtypes.
+Also supports batched inputs, and, if the input is batched, the output is batched with the same dimensions.
+
```

Comment:
cc @Lezcano before 1.9 we should probably tweak this to "see also" vector_norm and matrix_norm or even more directly tell people to prefer {vector, matrix}_norm

Reply:
Agree. You can have a look at the wording used in the `seealso` section of `inv` and `pinv` for inspiration on how to recommend a different function to the user:
https://github.com/pytorch/pytorch/blob/b60685c2d0b1786dbef80e111a382fb34746cf3f/torch/linalg/__init__.py#L121-L124